packages/css/src/components/field/field.scss

```diff
@@ -0,0 +1,13 @@
+/**
+ * @license EUPL-1.2+
+ * Copyright Gemeente Amsterdam
+ */
+
+.ams-field {
+  break-inside: avoid;
+}
+
+.ams-field__has-error {
```

Comment:
I think `has-error` is a modifier, not an element, so we need dashes here instead of the underscores.